stockticker/stockticker.py

```diff
@@ -40,7 +41,11 @@ def create_app(prophet_url, secret_key, quandl_key, bokeh_version): # pylint: di
     def index(): # pylint: disable=W0612
         """main route"""
         # Replace with Quandl API call on user input - may need to edit test
-        df = query_quandl('GOOGL', quandl_key) # pylint: disable=C0103
+        ticker = 'GOOGL' # place holder
+        if ticker in VALIDT:
```

Comment:
I would prefer to put this into something like a `validate_ticker` function which handles the logic of validating the ticker (and maybe returning a default one), this way we can test that function in a bit of an easier way.  We also might want to think about handling things like extra spaces in such a function.